src/MessagePack/MessagePack.csproj

```diff
@@ -1,10 +1,11 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0;netstandard2.1;net8.0;net472</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;netstandard2.1;net8.0;net9.0;net472</TargetFrameworks>
+    <LangVersion>13</LangVersion>
```

Comment:
Consider changing this in the repo root Directory.Build.targets instead of here so you get C# 13 in more projects.

Reply:
Root settings do affect Unity (creating situations where something works in Visual Studio but not in Unity).
I was surprised that CentralPackageTransitivePinningEnabled also affected Analyzers and broke the environment.
The recent increase in implicit insertions and high rate of problems is getting a bit tiresome.
Of course, many things have become more convenient.
That said, yes, let's move the LangVersion specification to Directory.Build.targets.

Reply:
> The recent increase in implicit insertions and high rate of problems is getting a bit tiresome.

Do you want to elaborate on this? What are these implicit insertions you speak of?

Reply:
I am saying that there are increasing cases that cannot be solved by looking at the csproj file alone.
Configuration settings are being inserted from various places.